docs/developing.md

```diff
@@ -244,14 +242,22 @@ By running the above-mentioned command, it will automatically set up all the pre
 
 #### Post Setup Requirements
 
-After you have done setting up pre-commit checks, then you may need to fulfill a few more requirements to finish setting up the development environment with `NixOS dev-shell`. These include:
+Inside `flake.nix`, the `$HOME/.cargo/bin` is added to the `PATH`. The git hook manager for this project is `rusty-hook`, and (unfortunately) it has to be installed via `cargo` and can't be neatly packaged up inside the flake ([see issue](https://github.com/swellaby/rusty-hook/issues/168)). This was automatically installed when you ran `cargo test`.
```

Comment:
```suggestion
```

I think I would suggest not including this part because I think the documentation should contain instruction which should be straight to the point. Also, specially developer documentation should be very straight to the point otherwise it gives a very bad impression.

For example:

here is a developer documentation from some project:

- https://github.com/neovim/neovim/blob/master/CONTRIBUTING.md
- https://github.com/Lissy93/dashy/blob/master/docs/developing.md
- https://github.com/libp2p/rust-libp2p/blob/master/docs/coding-guidelines.md

You can see that the developer documentation tends to be pretty straight to the point, with the explanations as required. That's why I would suggest not including it. :slightly_smiling_face:

Reply:
Thank you for the feedback, will get to work on this. 
Question:
- Do you prefer having new commits on the revision,
- or, do you want a "clean" git history where I would force push my revision,
- or, do you prefer that I write my revision here as comments so that you can give me some feedback before making a new commit?

Reply:
We would suggest having revisions as new commits that would be better because at the end of the day all PRs are squashed and merged so the git history always remains clean. :slightly_smiling_face: